build.gradle

```diff
@@ -95,7 +95,7 @@ subprojects {
   boolean isLint = project.path.endsWith("-lint")
   boolean isKotlin = project.path.endsWith("-ktx") || isLint || isMixedSourceSet
   boolean isSample = project.name == "sample"
-  boolean isJavaLibrary = !isAndroidLibrary && !isKotlin && !isSample || isMixedSourceSet
+  boolean isJavaLibrary = !isAndroidLibrary && !isKotlin && !isSample || (isMixedSourceSet && !isAndroidLibrary)
```

Comment:
> (isMixedSourceSet && !isAndroidLibrary)

Isn't `!isAndroidLibrary` redundant?

Reply:
The OR branch is always evaluated and if it's an Android library, we wouldn't want to apply the 'java-library` plugin (it gives an error).